mne/viz/backends/base_renderer.py

```diff
@@ -142,8 +142,9 @@ def sphere(self, center, color, scale, opacity=1.0,
 
     @abstractclassmethod
     def tube(self, origin, destination, radius=1.0, color=(1.0, 1.0, 1.0),
-             scalars=None, vmin=None, vmax=None, colormap=None,
-             opacity=1.0, backface_culling=False, reverse_lut=False):
+             scalars=None, vmin=None, vmax=None, colormap='RdBu',
+             normalized_colormap=False, opacity=1.0, backface_culling=False,
+             reverse_lut=False):
```

Comment:
can you avoid exposing option that we don't change in any examples?

Reply:
Following your advice, I decided to remove `opacity` and `backface_culling` for now.